assert/assertions.go

```diff
@@ -301,13 +301,17 @@ func messageFromMsgAndArgs(msgAndArgs ...interface{}) string {
 func indentMessageLines(message string, longestLabelLen int) string {
 	outBuf := new(bytes.Buffer)
 
-	for i, scanner := 0, bufio.NewScanner(strings.NewReader(message)); scanner.Scan(); i++ {
-		// no need to align first line because it starts at the correct location (after the label)
-		if i != 0 {
-			// append alignLen+1 spaces to align with "{{longestLabel}}:" before adding tab
+	msgScanner := bufio.NewScanner(strings.NewReader(message))
+	msgScanner.Buffer([]byte{}, len(message)+1)
```

Comment:
Set the buffer to `nil` instead of a useless zero-length buffer (`[]byte{}`) that will have to be reallocated in all cases.

```go
msgScanner.Buffer(nil, len(message)+1)
```